website/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Triphouse project</title>
+</head>
+<body>
+    <div class="container">
+        <div class="first_block">
+            <h1>What do we offer</h1>
+            <div class="line"></div>
+        </div> 
+        <div class="item_container">
+            <div class="item">
+                <img class="item_icon" src="img/Support.png">
+                <div class="item_title">Support 24/7</div> 
+            </div>
+            <div class="item">
+                <img class="item_icon" src="img/Communicate.png">
+                <div class="item_title">Communicate directly</div> 
+            </div>
+            <div class="item">
+                <img class="item_icon" src="img/Book.png">
+                <div class="item_title">Book online</div> 
+            </div>
+            <div class="item">
+                <img class="item_icon" src="img/Real guest.png">
+                <div class="item_title">Real guest reviews</div> 
+            </div>
```

Comment:
To make text on the one level you can add a div wrapper for the images and set the height to match the height of the tallest image.